load.php

```diff
@@ -197,3 +197,47 @@ function perflab_load_active_modules() {
 if ( is_admin() ) {
 	require_once plugin_dir_path( __FILE__ ) . 'admin/load.php';
 }
+
+// WP Filesize function.
+if ( ! function_exists( 'wp_filesize' ) ) {
+	/**
+	 * Wrapper for PHP filesize with filters and casting the result as an integer.
+	 *
+	 * This function is introduced in WP 6.0, to support backward compatibility
+	 * function is added as backup here.
+	 *
+	 * @param string $path Path to the file.
+	 *
+	 * @return int The size of the file in bytes, or 0 in the event of an error.
+	 * @since n.e.x.t
+	 *
+	 * @link https://www.php.net/manual/en/function.filesize.php
+	 */
+	function wp_filesize( $path ) {
```

Comment:
@mehulkaklotar Can we move this function to a separate file? Something like `polyfills.php` and include that file here. That will make it easier to add/remove polyfills as we bump versions.

Reply:
Fixed! thanks @akshitsethi